http-client/src/main/java/io/airlift/http/client/Request.java

```diff
@@ -316,6 +330,18 @@ public Builder setIdleTimeout(Duration timeout)
             return this;
         }
 
+        public Builder setMaxContentLength(Optional<DataSize> maxContentLength)
```

Comment:
Do we need this method? We don't have it for `setVersion()`. It's fine if we need it.